.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -373,15 +374,20 @@ jobs:
 
       - name: Upload Artifacts (DevOps GCP Bucket)
         uses: google-github-actions/upload-cloud-storage@386ab77f37fdf51c0e38b3d229fad286861cc0d0 # v2.2.1
-        if: ${{ inputs.dry-run-enabled != true && !cancelled() && !failure() }}
+        if: ${{ inputs.dry-run-enabled != true &&
+          !cancelled() &&
+          !failure() }}
         with:
           path: ${{ steps.artifact-release.outputs.folder }}
           destination: ${{ secrets.bucket-name }}
           parent: false
 
       - name: Upload Artifacts (CDN Bucket)
         uses: google-github-actions/upload-cloud-storage@386ab77f37fdf51c0e38b3d229fad286861cc0d0 # v2.2.1
-        if: ${{ inputs.dry-run-enabled != true && needs.validate.outputs.prerelease != 'true' && !cancelled() && !failure() }}
+        if: ${{ inputs.dry-run-enabled != true &&
+          needs.validate.outputs.prerelease != 'true'&&
+          !cancelled() &&
+          !failure() }}
```

Comment:
Does this exceed a single line?

Reply:
I collapsed it into a single line and it looks fine.